### PR TITLE
Electrons hazelhen

### PR DESCRIPTION
**Pull request by alhom**
Static ionosphere implementation for TEMPO electrons, as a new sysboundary type.

### CODE REVIEW

File: sysboundary/staticionosphere.cpp [hunk 1]
```diff
@@ -0,0 +1,811 @@
...
+      // Per-population parameters
+      for(uint i=0; i< getObjectWrapper().particleSpecies.size(); i++) {
+         const std::string& pop = getObjectWrapper().particleSpecies[i].name;
+         
+         Readparameters::add(pop + "_staticionosphere.taperRadius", "Width of the zone with a density tapering from the ionospheric value to the background (m)", 0.0);
+         Readparameters::add(pop + "_staticionosphere.rho", "Number density of the ionosphere (m^-3)", 1.0e6);
+         Readparameters::add(pop + "_staticionosphere.VX0", "Bulk velocity of ionospheric distribution function in X direction (m/s)", 0.0);
+         Readparameters::add(pop + "_staticionosphere.VY0", "Bulk velocity of ionospheric distribution function in X direction (m/s)", 0.0);
+         Readparameters::add(pop + "_staticionosphere.VZ0", "Bulk velocity of ionospheric distribution function in X direction (m/s)", 0.0);
+         Readparameters::add(pop + "_staticionosphere.fluffiness", "Inertia of boundary smoothing when copying neighbour's moments and velocity distributions (0=completely constant boundaries, 1=neighbours are interpolated immediately).", 0);
```
Comment (markusbattarbee) [inline sysboundary/staticionosphere.cpp]: Since it's static, fluffiness is not relevant.

**Review by markusbattarbee — CHANGES_REQUESTED** [review]
Just the fluffiness thing out

**Comment by markusbattarbee** [comment]
Tested to work with the getnormalvector out? I think that was indeed purged from fieldsolver stuff some while back, just checking

**Comment by alhom** [comment]
It does not appear anywhere else than ionosphere class, so it shouldn't matter. Let me verify that, though, I'm not sure if I ran that with actual ionosphere in the domain.

**Comment by alhom** [comment]
Works without the getnormalfield function.

**Pull request merged**